app/src/main/AndroidManifest.xml

```diff
@@ -13,7 +13,9 @@
         android:theme="@style/AppTheme">
         <activity
             android:name=".activity.MainActivity"
-            android:label="@string/app_name">
+            android:label="@string/app_name"
+            android:launchMode="singleTop"
```

Comment:
used to preserve the `MainActivity` instance on navigation to/from the `DetailActivity`